test/RoaveTest/SecurityAdvisories/VersionConstraintTest.php

```diff
@@ -375,8 +360,8 @@ public function mergeableRangesProvider() : array
             ['>1,<2.10', '>1,<2.100', false, true],
             ['>1.0,<2', '>1,<2', true, true],
             ['>1,<2.0', '>1,<2', true, true],
-            ['>1.0.0,<2', '>1,<2', true, true],
-            ['>1,<2.0.0', '>1,<2', true, true],
+            ['>1.0.0,<2', '>1.0.0 ,<2', true, true],
```

Comment:
These should be additions, not replacements. The `'>1,<2'` should still be checked

Reply:
you see, when I add version I'm not deleting trailing zeroes. 

Zeroes are deleted only when we finally assemble versions for composer.json file. So 378 and 379 are not working with current logic. What to do in this case?